db/schema.rb

```diff
@@ -72,17 +72,6 @@
     t.datetime "attachment_updated_at"
   end
 
-  create_table "donations", force: :cascade do |t|
-    t.string   "first_name"
-    t.string   "last_name"
-    t.string   "email"
-    t.decimal  "amount",     precision: 10, scale: 2
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
-    t.text     "comment"
-    t.string   "uid"
-  end
```

Comment:
Is there a reason you got rid of donations? Make sure it's still in your local DB, should be there still I think